### ISSUE TITLE
Close or transfer this cookbook

### ISSUE THREAD

**Issue by jldugger** [osuosl-cookbooks/pgd#24]
We should transfer this repo to its proper owner, where it may be of value.

**Comment by mathuin** [comment]
This is a good idea, along with the osuosl/pgd repository, once a proper owner has been identified.

**Comment by jldugger** [comment]
Yep, I just wanted to create a reminder to do this as I remove it from mrconfig and such.
